includes/tracks/events/class-wc-order-tracking.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * WooCommerce Order Tracking
+ *
+ * @package WooCommerce\Tracks
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * This class adds actions to track usage of a WooCommerce Order.
+ */
+class WC_Order_Tracking {
+	/**
+	 * Init tracking.
+	 */
+	public function init() {
+		add_action( 'woocommerce_admin_order_data_after_order_details', array( $this, 'track_order_viewed' ) );
+	}
+
+	/**
+	 * Send a Tracks event when an order is viewed.
+	 *
+	 * @param object $order Order.
+	 */
+	public function track_order_viewed( $order ) {
+		$properties = array(
+			'current_status'   => $order->get_status(),
+			'date_created'     => $order->get_date_created(),
```

Comment:
`$order->get_date_created()` returns an object with several properties, which all get supplied in the payload.

<img width="301" alt="Screen Shot 2020-03-27 at 2 20 03 PM" src="https://user-images.githubusercontent.com/1922453/77711689-80d1cf00-7036-11ea-8c1a-27d22de36224.png">

Maybe just the date is needed?

Reply:
Done, I formatted it as an ISO8661 string.